x-pack/plugins/metrics_data_access/common/http_api/metrics_api.ts

```diff
@@ -30,7 +19,7 @@ export const MetricsAPIMetricRT = rt.type({
 
 export const MetricsAPIRequestRT = rt.intersection([
   rt.type({
-    timerange: MetricsAPITimerangeRT,
+    timerange: timeRangeRT,
```

Comment:
I saw in the metric threshold rule document, that there is this option to [specify timestamp](https://www.elastic.co/guide/en/observability/current/metrics-threshold-alert.html#metrics-alert-settings) which I didn't see it in the setting, did we have this setting at some point?

![image](https://github.com/elastic/kibana/assets/12370520/b9951b8a-3bbe-4bab-b1fb-b6bab0fa2648)